src/Psl/Filesystem/copy.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Filesystem;
+
+use Psl;
+use Psl\Internal;
+use Psl\Str;
+
+use function fclose;
+use function fopen;
+use function stream_copy_to_stream;
+
+/**
+ * Change the group ownership of $filename.
+ *
+ *
+ * @throws Exception\RuntimeException                   If unable to change the group ownership for $filename.
+ * @throws Psl\Exception\InvariantViolationException    If $source does not exist or is not readable.
+ */
+function copy(string $source, string $destination, bool $overwrite = false): void
```

Comment:
It might have been better if you had IO + Filehandle ready to use inside this function.

Reply:
yea, this is currently just for local files, i think it's okay to keep it even if we have `IO\copy(ReadHandle $source, WriteHandle $destination): void`